production/helm/loki/values.yaml

```diff
@@ -656,7 +658,7 @@ monitoring:
       name: "self-monitoring"
       # -- Namespace to create additional tenant token secret in. Useful if your Grafana instance
       # is in a separate namespace. Token will still be created in the canary namespace.
-      secretNamespace: "{{ .Release.Namespace }}"
+      secretNamespace: "{{ default .Release.Namespace *namespaceOverride }}"
```

Comment:
I'm 100% confident that YAML anchors doesn't work in Go Templates. YAML anchors only works inside YAML documents